oat-sa/extension-tao-itemqti
# Fix/aut-1979/handle-imported-items

related to https://oat-sa.atlassian.net/browse/AUT-1979

### Description

If item is authored using Terre feature set and then imported into authoring having Solar feature set, some of features are still visible in ui

https://oat-sa.atlassian.net/browse/AUT-1979?focusedCommentId=188272

This PR tend to fix :

 - Presence of shuffle buttons in UI of various interactions using SimpleChoice / InlineChoice component
 - Tooltip is displayed in the imported item with set-up tooltip

### How to test

- Setup authoring to Terre feature set 
(run command  
`php index.php 'oat\tao\scripts\tools\FeatureFlag\FeatureFlagTool' -s FEATURE_FLAG_TAO_ADVANCE_ONLY -v false`
in fpm container and `redis-cli flushall` in redis container)

 - Create item with ChoiceInteraction, InlineChoice interaction, OrderInteraction, Match and enable choice shuffle

 - Create various interactions which have tooltips in prompts, A-blocks, choices
 
 - Setup authoring to Solar feature set (run command  
`php index.php 'oat\tao\scripts\tools\FeatureFlag\FeatureFlagTool' -s FEATURE_FLAG_TAO_ADVANCE_ONLY -v true`
in fpm container and `redis-cli flushall` in redis container)

 - check the interactions have no shuffle buttons and no tooltips which you have set up
